account/identity.go

```diff
@@ -140,10 +142,18 @@ func (m *GormIdentityRepository) Create(ctx context.Context, model *Identity) er
 	}
 	err := m.db.Create(model).Error
 	if err != nil {
-		goa.LogError(ctx, "error adding Identity", "error", err.Error())
+		log.LogError(ctx, map[string]interface{}{
+			"identity": model,
+			"err":      err.Error(),
+		}, "Unable to create the identity")
 		return errors.WithStack(err)
```

Comment:
We should probably replace this line with:

```go
errors.Wrap(err, "unable to create the identity")
```

The use of functions from the `github.com/pkg/errors` package should allow the caller to find out about the stack or the original cause of an error. On the other hand we don't know nothing about the item that `model` object you're logging then.